### ISSUE TITLE
README.md should provide sensible advice on ExecutionPolicy

### ISSUE THREAD

**Issue by cgranade** [cgranade/posh-tex#2]
Installers and arXiv manifests that use PoShTeX currently can cause a lot of confusion due to the subtleties of ``Set-ExecutionPolicy``. Since getting LaTeX package authors to also sign their installers is probably out of scope, a reasonable approach might be to remind users that ``Unblock-File`` will clear any metadata about a PoShTeX-based installer that prevents its execution under ``RemoteSigned``.
